### PR TITLE
updated kube-cluster (0.1.9)

### PR DESCRIPTION
**Pull request by geoHeil**

### CODE REVIEW

File: Casks/kube-cluster.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'kube-cluster' do
-  version '0.1.8'
-  sha256 'e15105317cf138f47716108212d4ffba1ae6a48088d345dc67ea20d159c13efa'
+  version '0.1.9'
+  sha256 '3616bffc8a553a2dd67ab416f7c067588063cbd4c7a99b5c9aa2fa272a4181b0'
 
   url "https://github.com/TheNewNormal/kube-cluster-osx/releases/download/v#{version}/Kube-Cluster_v#{version}.dmg"
   appcast 'https://github.com/TheNewNormal/kube-cluster-osx/releases.atom',
```
Comment (vitorgalvao) [inline Casks/kube-cluster.rb]: Updating the `:checkpoint` value in `appcast` is required. You can find it with `curl --silent --compressed "{{appcast_url}}" | sed 's|<pubDate>[^<]*</pubDate>||g' | shasum --algorithm 256 | awk '{ print $1 }'` (`cask-repair` also does it).

**Comment by geoHeil** [comment]
No. Your script only outputs this: 
and the checkpoint seems to be unchanged.
```
@@ -1,6 +1,6 @@
 cask 'kube-cluster' do
-  version '0.1.8'
-  sha256 'e15105317cf138f47716108212d4ffba1ae6a48088d345dc67ea20d159c13efa'
+  version '0.1.9'
+  sha256 '3616bffc8a553a2dd67ab416f7c067588063cbd4c7a99b5c9aa2fa272a4181b0'

   url "https://github.com/TheNewNormal/kube-cluster-osx/releases/download/v#{version}/Kube-Cluster_v#{version}.dmg"
   appcast 'https://github.com/TheNewNormal/kube-cluster-osx/releases.atom',
--------------------------------------------------------------------------------
Is everything correct? You can also make further manual edits (y/n/e)
```

The code you kindly provided return this: `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` I will update checkpoint

**Comment by vitorgalvao** [comment]
You were right, it was unchanged, please revert it back to what you had, as it was correct.

Regarding the command I gave you, `{{appcast_url}}` was meant to be substituted. No matter. I’ll grab your submission and throw away the `updated checkpoint` commit.

Thank you for sticking with us through the submission.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0d1e0b2577cf42f217f9fafcafac80f2c00db1ca. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**